fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -68686,6 +68664,161 @@ relationship: overlaps FBbt:00007053 ! lateral horn
 created_by: CP
 creation_date: 2018-06-04T15:25:40Z
 
+[Term]
+id: FBbt:00047216
+name: larval DMS cell of pars intercerebralis
+def: "Neuron of the larval pars intercerebralis (PI) that expresses dromyosuppressin (DMS) as well as dimmed. There are two of these cells per hemisphere. These cells are distinct from the PI neurons that express insulin like peptides, diuretic hormone 44 and SIFamide." [FlyBase:FBrf0210278]
+synonym: "PI 13-14" EXACT [FlyBase:FBrf0210278]
+relationship: capable_of GO:0030072 ! peptide hormone secretion
+relationship: expresses http://flybase.org/reports/FBgn0011581
+relationship: has_soma_location FBbt:00111670 ! larval pars intercerebralis
+created_by: CP
+creation_date: 2018-06-05T15:13:30Z
+
+[Term]
+id: FBbt:00047217
+name: larval SIFamide cell of pars intercerebralis
+def: "Neuron of the larval pars intercerebralis (PI) that expresses SIFamide (SIFa) as well as dimmed. There are two of these cells per hemisphere. These cells are distinct from the PI neurons that express insulin like peptides, diuretic hormone 44 and dromyosuppressin." [FlyBase:FBrf0210278]
+synonym: "PI 8-9" EXACT [FlyBase:FBrf0210278]
+relationship: capable_of GO:0030072 ! peptide hormone secretion
+relationship: expresses http://flybase.org/reports/FBgn0053527
+relationship: has_soma_location FBbt:00111670 ! larval pars intercerebralis
+created_by: CP
+creation_date: 2018-06-06T08:14:14Z
+
+[Term]
+id: FBbt:00047229
+name: ILP7 neuron
+def: "ILP7-expressing neuron. All identified ILP7 neurons are found in the abdominal segment of the larval ventral nerve cord or adult thoracico-abdominal ganglion. There is one dorsal pair in the A1 (abdominal 1) neuromere, one lateral pair in each of A1-4 and a posterior group (Miguel-Aliaga et al., 2008; Castellanos et al., 2013)." [FlyBase:FBrf0204880, FlyBase:FBrf0222499]
+intersection_of: FBbt:00005106 ! neuron
+intersection_of: expresses http://flybase.org/reports/FBgn0044046
+relationship: capable_of GO:0046879 ! hormone secretion
+created_by: CP
+creation_date: 2018-06-12T08:48:30Z
+
+[Term]
+id: FBbt:00047230
+name: embryonic/larval ILP7 neuron
+def: "ILP7 neuron of the embryo or larva. These are found in the abdominal neuromeres of the ventral nerve cord. By third instar stage, there are two lateral ILP7 neurons in each of abdominal (A) neuromeres 1-4, a dorsal pair in A1, and two posterior neurons in each of neuromeres A6-A9." [FlyBase:FBrf0204880]
+is_a: FBbt:00005130 ! neurosecretory neuron
```

Comment:
Perhaps move this classification up to ILP7 neuron.

Reply:
This is already inferred for 'ILP7 neuron' - removed assertion from subclasses